picard/ui/matchdetailsdialog/matchdetails.py

```diff
@@ -0,0 +1,211 @@
+            if json is not None:
+                self.set_json_edit(json)
+            if file_ is not None:
+                self.set_input_filename_lbl(file_.filename)
+            if file_ is not None:
+                self.set_acoustid_lookup_url_edit(fingerprint_lkp_url)
+            self.current_file = file_
+            self.prepare_table()
+            self.entries = match_details_log.get_match_dtls(file_.filename)
+            for row, entry in enumerate(self.entries):
+                self.table.insertRow(row)
+                self.set_table_item(row, 'nbr',                  row)
+                self.set_table_item(row, 'match_dtl_entry_type', entry.match_dtl_entry_type)
+                self.set_table_item(row, 'is_new',               entry.is_new_str)
+                self.set_table_item(row, 'similarity',           entry.similarity)
+                self.set_table_item(row, 'acoust_id',            entry.acoust_id)
+                self.set_table_item(row, 'rec_id',               entry.rec_id)
+                self.set_table_item(row, 'rec_title',            entry.rec_title)
+                self.set_table_item(row, 'rec_len',              entry.rec_len_str)
+                self.set_table_item(row, 'rec_src_cnt',          entry.rec_src_cnt)
+                self.set_table_item(row, 'rec_artist',           entry.rec_artist)
+                self.set_table_item(row, 'rg_id',                entry.rg_id)
+                self.set_table_item(row, 'rg_title',             entry.rg_title)
+                self.set_table_item(row, 'rg_pri_type',          entry.rg_pri_type)
+                self.set_table_item(row, 'rg_sec_type',          str(entry.rg_sec_type))
+                self.set_table_item(row, 'rel_id',               entry.rel_id)
+                self.set_table_item(row, 'rel_ctry',             entry.rel_ctry)
+                self.set_table_item(row, 'rel_dt',               entry.rel_dt)
+                self.set_table_item(row, 'med_fmt',              entry.med_fmt)
+                self.set_table_item(row, 'med_pos',              entry.med_pos)
+                self.set_table_item(row, 'med_tot_cnt',          entry.med_tot_cnt)
+                self.set_table_item(row, 'trk_id',               entry.trk_id)
+                self.set_table_item(row, 'trk_pos',              entry.trk_pos)
+                self.set_table_item(row, 'med_trk_cnt',          entry.med_trk_cnt)
+                self.set_table_item(row, 'rel_tot_trk_cnt',      entry.rel_tot_trk_cnt)
+                self.set_table_item(row, 'parts',                str(entry.parts))
+                self.set_table_item(row, 'release_parts',        str(entry.release_parts))
+                self.set_table_item(row, 'recording_base_score', entry.recording_base_score)
+
+            self.show_table(sort_column='nbr', sort_order=QtCore.Qt.AscendingOrder)
+
+        except TypeError as e:
+            log.warning("MATCH:No log entries for " + file_.filenamei + " due to " + e)
```

Comment:
`log.warning("MATCH:No log entries for %s due to %s" , file_.filenamei, e)`

Reply:
Made more sense to me when I had the catchall, so now I removed it.